src/commands/src/scripts/local-build/utils/get-env-vars.ts

```diff
@@ -0,0 +1,174 @@
+import { SSMClient, GetParameterCommand } from '@aws-sdk/client-ssm';
+
+// This array contains the Parameter Store paths of the secure strings
+// we want to add to the worker environment. These are mapped
+// to their environment variable name in the workerParamPathToEnvName map.
```

Comment:
This is correct. I think that's a great idea, but I am not sure of the best way to go about it since the `cdk-infra` repository is separate from the Autobuilder code. I have had issues trying to import stuff outside of `cdk-infra`, but maybe it would be okay for the Autobuilder to import functionality from `cdk-infra`? This is something worth discussing I think with the team more generally of how we want to share stuff in a monorepo environment. 

I've also had this question with respect to types that are shared throughout different parts of the project.

Reply:
Oh, gotcha. I wasn't aware that importing stuff from outside of `cdk-infra` was an issue. Is that just a limitation of how CDK works/builds things?

Also wanted to note that this comment isn't necessarily a blocker for this PR. It's just a quality-of-life concern for us. So if there isn't an easy way to resolve this immediately, that's okay

Reply:
I think it's more of a limitation of a TypeScript/node project's ability to import stuff outside of the given root project directory. I guess though, we should have the ability to grab these functions from `/cdk-infra` since it is a part of the `docs-worker-pool` root directory. 

To be honest, this was done in an expedient fashion haha. I wanted to get it working, but knew this was less than ideal essentially copying and pasting stuff. I think it's worth flagging this though and creating a ticket to refactor this. One common pain point lately has been having to do duplicate work, and this definitely introduces more of that.

Reply:
Makes sense. Thanks!